Sources/Nakama/Mappings.swift

```diff
@@ -333,3 +352,16 @@ extension Nakama_Realtime_UserPresence {
         )
     }
 }
+
+extension Nakama_Realtime_Party {
+    func toParty() -> Party {
+        return Party(
+            id: self.partyID,
+            open: self.open,
+            maxSize: Int(self.maxSize),
+            self_p: self.self_p.toUserPresence(),
+            Leader: self.leader.toUserPresence(),
```

Comment:
is this meant to be uppercase?

Reply:
Yeah because self is a reserved keyword.

Reply:
Right, Leader should be lowercase.